deploy/018-native-token-gateway.ts

```diff
@@ -0,0 +1,25 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { getVWNativeToken } from "../helpers/deploymentConfig";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const vWNativeAddress = getVWNativeToken(hre.network.name);
+
+  await deploy("NativeTokenGateway", {
+    contract: "NativeTokenGateway",
```

Comment:
I would add to the name of the NativeTokenContract the symbol of the vToken associated, in a dynamic way: 
```
contract: `NativeTokenGateway_${vToken.name}`
```
This would be helpful if we have more than one market for WETH or WBNB, for example

Reply:
Maybe pool name? A single pool can have only one market for an asset.

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/361/commits/b508cb1ea7a7fd40bba305ac47aca94f903aae9f